provider/resources.go

```diff
@@ -1269,8 +1272,12 @@ func Provider() tfbridge.ProviderInfo {
 			"aws_ec2_local_gateway_route_table_vpc_association": {
 				Tok: awsResource(ec2Mod, "LocalGatewayRouteTableVpcAssociation"),
 			},
-			"aws_ec2_tag":                       {Tok: awsResource(ec2Mod, "Tag")},
-			"aws_dedicated_host":                {Tok: awsResource(ec2Mod, "DedicatedHost")},
+			"aws_ec2_tag": {Tok: awsResource(ec2Mod, "Tag")},
+			"aws_dedicated_host": {
+				Tok:                awsResource(ec2Mod, "DedicatedHost"),
+				DeprecationMessage: "This resource has been deprecated in favour of `aws.ec2.Ec2DedicatedHost`",
+			},
+			"aws_ec2_host":                      {Tok: awsResource(ec2Mod, "Ec2DedicatedHost")},
```

Comment:
Why not just `Host`?

Reply:
We have decided to be able to use the DedicatedHost name here as the APIs are identical moving between versions of the resource

Reply:
That's the best outcome then 👍